ngraph/core/include/ngraph/op/depth_to_space.hpp

```diff
@@ -72,6 +73,8 @@ namespace ngraph
 
                 virtual std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
+                bool evaluate(const HostTensorVector& outputs,
```

Comment:
Please inherit from simple op, not fused op and remove decomposition completely

Reply:
BTW, should we add ITT instrumentation macro for all new evaluates?

Reply:
Fixed